client/src/Pages/SignUpPage.jsx

```diff
@@ -19,19 +19,75 @@ const styles = theme => ({
         display: 'flex',
         justifyContent: 'center',
         alignItems: 'center',
+        backgroundColor: grey[100],
     },
     panel: {
         display: 'flex',
         flexDirection: 'column',
-        width: '400px',
-        backgroundColor: grey[100],
+        width: '450px',
+        backgroundColor: '#fff',
         border: `1px solid ${grey[300]}`,
-        padding: 15,
+        padding: '40px 15px',
         borderRadius: 5,
     },
+    title: {
+        color: blue[500],
+        fontWeight: 'bold',
+    },
     loginLink: {
-        alignSelf: 'flex-end',
-    }
+        alignSelf: 'center',
+        textDecoration: 'none',
+        color: blue[500],
+        textTransform: 'uppercase',
+        fontFamily: '"Roboto", "Helvetica", "Arial", sans-serif'
+    },
+    profile: {
+        position: 'relative',
+    },
+    hr: {
+        backgroundColor: grey[300],
+        border: 'none',
+        height: 1,
+        width: '100%',
+        margin: '20px 0'
+    },
+    profileHr: {
+        backgroundColor: grey[300],
+        border: 'none',
```

Comment:
I see same styles for SignInPage, can you put it in separate file and reuse for both pages?